### ISSUE TITLE
Add standard way to get keyboard focus to current pane's toolbar

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6543]
In RStudio 1.3, there's a command / shortcut to get focus to the main window toolbar (Help / Accessibility / Focus / Focus Main Toolbar.

Need a similar mechanism to get focus to the current pane's toolbar(s), othewise this is very difficult to do and we have quite a bit of toolbar-only functionality (contextual commands that don't always show up in the main menus).

Ideally, this should be a single "Focus current pane's toolbar" command that will leverage work to have workbench be more aware of what pane has focus, see also #6225 and #6073.

**Comment by gtritchie** [comment]
Moving out of 1.4. Really want it, but requires a bunch of infrastructure work likely needing to wait to 1.5.

**Comment by gtritchie** [comment]
Lower priority for accessibility relative to others, moving to 1.6.
